### PR TITLE
Update REST models to use getFullName

### PR DESCRIPTION
**Pull request by ijungmann**
**Description**:
Update the REST model classes to use getFullName() for getting tableAlias.columnName, as opposed to having a long list of additional COLUMN_FULL_NAME attributes.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — APPROVED** [review]
Was hoping there was a way to share the single method, but probably not worth the code churn for now.
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

File: hedera-mirror-rest/model/assessedCustomFee.js [hunk 1]
```diff
@@ -36,15 +36,20 @@ class AssessedCustomFee {
...
   static AMOUNT = `amount`;
-  static AMOUNT_FULL_NAME = `${this.tableAlias}.${this.AMOUNT}`;
   static COLLECTOR_ACCOUNT_ID = `collector_account_id`;
-  static COLLECTOR_ACCOUNT_ID_FULL_NAME = `${this.tableAlias}.${this.COLLECTOR_ACCOUNT_ID}`;
   static CONSENSUS_TIMESTAMP = `consensus_timestamp`;
-  static CONSENSUS_TIMESTAMP_FULL_NAME = `${this.tableAlias}.${this.CONSENSUS_TIMESTAMP}`;
   static EFFECTIVE_PAYER_ACCOUNT_IDS = `effective_payer_account_ids`;
-  static EFFECTIVE_PAYER_ACCOUNT_IDS_FULL_NAME = `${this.tableAlias}.${this.EFFECTIVE_PAYER_ACCOUNT_IDS}`;
   static TOKEN_ID = `token_id`;
-  static TOKEN_ID_FULL_NAME = `${this.tableAlias}.${this.TOKEN_ID}`;
```
Comment (steven-sheehy) [inline hedera-mirror-rest/model/assessedCustomFee.js:47]: Down side is we're now no longer caching those string as constants and have to re-create them every time. If we could move the method to a base class in the future we could have a static map that stores it.

**Review by steven-sheehy — APPROVED** [review]

File: hedera-mirror-rest/model/assessedCustomFee.js (hunk 1 above)
Comment (xin-hedera) [reply hedera-mirror-rest/model/assessedCustomFee.js:47]: we can do it. there's probably the drawback that IDE can't find these symbols.

**Pull request merged**